src/modules/integrator-guide/pages/compose-configuration/duplication-detection.adoc

```diff
@@ -3,45 +3,98 @@ include::ROOT:partial$variables.adoc[]
+The duplicate detection feature is enabled by using validation duplicate detection rules on a module.
+The rules ensure that no duplicate values are recorded while creating or modifying a record by ensuring that the designated fields do not have the same values.
 
-Validation can be configured using two rules:
-
-* Strict duplicate value validation
+Duplicate detection validation can be configured using two rules:
 
+[cols="1s,5a"]
+|===
+| [#Strict-duplicate-validation]#<<Strict-duplicate-validation,Strict duplicate validation rule>>#
+|
 This rule ensures that a record will not be saved when a duplicate value is detected in the selected fields in any existing record of this module. 
 
-When the Strict duplicate value validation rule is applied, an error message pop-up will be displayed on the bottom right side of the screen as an indication that a record with the same values exists.
+When the Strict duplicate value validation rule is applied, a pop-up error message indicating the existence of a record with identical values will appear in the bottom right corner of the screen..
 
-* Soft duplicate value validation
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "duplication-detection/error.png",
+    "alias": "error",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 152,
+    "y": 42,
+    "w": 1568,
+    "h": 1080
+  },
+  "focus": {
+    "padding": "xs",
+    "x": 834,
+    "y": 450,
+    "w": 1204,
+    "h": 586
+  },
+  "annotations": []
+}
+----
 
-This rule allows a record to be saved but issues a warning message pop-up when a duplicate value is detected, indicating that there is an existing record with the exact field values within the module.
 
-When Soft duplicate value validation is applied, the warning message is displayed on the bottom right side of the screen.
+| [#Soft-duplicate-validation]#<<Soft-duplicate-validation,Soft duplicate validation rule>>#
+|
+This rule allows the saving of a record but displays a warning message when a duplicate value is identified, indicating that a record with identical field values already exists within the module.
 
-=== Enabling value validation rules:
+When Soft duplicate value validation is applied, the pop-up warning message is displayed on the bottom right side of the screen.
 
-[NOTE]
-====
-Validation rules can be enabled when creating new modules or on exiting modules.
-====
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "duplication-detection/warning.png",
+    "alias": "warning",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {
+    "x": 152,
+    "y": 42,
+    "w": 1568,
+    "h": 1080
+  },
+  "focus": {
+    "padding": "xs",
+    "x": 834,
+    "y": 450,
+    "w": 1204,
+    "h": 586
+  },
+  "annotations": []
+}
+----
+|
+|===
 
-Navigate to your Low Code namespace. Click on an existing module or create a new module.
+=== Enabling duplication detection validation rules:
```

Comment:
Consider making this a `==`; This is what I'd do since we don't have many sub titles and it (IMO) looks strange like this

![image](https://user-images.githubusercontent.com/12545711/198587103-c5fa2915-6f00-46d8-8eda-7ef50c1aac1f.png)